lib/foreman/renderer/scope/variables/base.rb

```diff
@@ -75,6 +75,7 @@ def kickstart_attributes
             @arch      = architecture_name
             @osver     = major.try(:to_i)
             @mediapath = mediumpath(@medium_provider) if @medium_provider
+            @repos     = repos(host)
```

Comment:
This will put the deprecation warning into logs every time we render a provision template. Not sure if this is what we want. Opinions? I suggest to comment the warning line out but keep it in the codebase including the YAML file so we are notified in the future to remove it.

Reply:
The only way a user would know then would be the release notes, since the logs won't tell them.  I think that's fine since I really doubt anyone uses this.  But, since @tbrisker asked for the deprecation maybe he has an opinion about it.

Reply:
I agree that having the warning every time a template is rendered is too much noise. I'm trying to figure out if there is some way of only logging that when the variable is accessed, but so far no luck. Perhaps we could show the deprecation only if the function isn't called from here? i.e. `Foreman::Deprecation ... unless caller.any? {|c| c.include? 'scope/variables/base'}`

Reply:
I am for comment only, this is not widely used variable I think.

Reply:
Updated to comment only.  Left the item in the whitelist, so we remember to remove it in 1.22. I don't think firing the message based only on other callers is helpful, if anyone uses this it'd be through the @repos variable.

Reply:
OK, i'm fine with this. Thanks @stbenjam and @lzap !